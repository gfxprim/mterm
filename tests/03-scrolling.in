10 5
a\n
b\n
c\n
d\n
e\n
Hello!
