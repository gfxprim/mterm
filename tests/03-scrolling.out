 ----------
|b         |
|c         |
|d         |
|e         |
|Hello!    |
 ----------
size 5x10 cursor 4x6
